client/my-sites/plan-features-2023-grid/style.scss

```diff
@@ -370,6 +398,13 @@
 		}
 	}
 
+	.plan-features-2023-grid__desktop-view,
+	.plan-features-2023-grid__tablet-view {
+		.plan-features-2023-grid__table-item {
+			max-width: 344px;
```

Comment:
`NB:`

Should we use `$table-cell-max-width` here?

Reply:
Of course, we could.